doc/BGP/BGP-route-aggregation-with-bbr-awareness.md

```diff
@@ -175,10 +193,14 @@ For every aggregated address, we track its state in state DB, it has two states
     ...
     "BGP_AGGREGATE_ADDRESS": {
         "192.168.0.0/24": {
-            "state": "inactive"
+            "state": "inactive",
+            "aggregate-address-prefix-list": "AGG_ROUTES_V4",
+            "contributing-address-prefix-list": "AGG_CONTRIBUTING_ROUTES_V4"
         },
         "fc00::/63": {
-            "state": "active"
+            "state": "active",
+            "aggregate-address-prefix-list": "None",
```

Comment:
Should we use **empty string** to indicate there is no need to add the prefix to list?

Reply:
Sure, no problem

Reply:
Updated